packages/calypso-color-schemes/package.json

```diff
@@ -25,7 +25,7 @@
 	},
 	"scripts": {
 		"clean": "npx rimraf js css src/__color-studio",
-		"prepare": "node bin/prepare-sass-assets.js && node bin/build-css.js"
+		"postinstall": "node bin/prepare-sass-assets.js && node bin/build-css.js"
```

Comment:
This might not be an exact replacement, as `prepare` would be run as part of the publish workflow, but `postinstall` might not be. 🤔 Do we also want to run this during `prepublishOnly`?

Reply:
Very good point.

Actually, this can't be `postinstall`, as the script will run for any consumer as soon as they do `yarn install @automattic/calypso-color-schemes` but the scripts will fail as they don't have the devDependencies required to run them.

I'll think a different way to do this. Maybe we need to do `yarn workspaces focus @automattic/calypso-color-schemes && yarn workspace @automattic/calypso-color-schemes prepare && yarn workspaces focus <app>` ?